app/src/main/res/drawable-v21/ic_shuffle_albums_scaled_24dp.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This icon is quite dense, and doesn't fit in too well next to the other icons, IMO. I'll see if I can come up with an alternative.

Reply:
By all means. This was my first ever experience with designing vector graphics and using Inkscape.